components/select/multiselect.js

```diff
@@ -0,0 +1,94 @@
+import Select from 'react-select'
+
+const selectOptions = [
+  { value: 'angellist', label: 'AngelList' },
+  { value: 'blog', label: 'Blog' },
+  { value: 'event', label: 'Event' },
+  { value: 'linkedin', label: 'LinkedIn' },
+  { value: 'linkedin_connection', label: 'LinkedIn Connection' },
+  { value: 'twitter', label: 'Twitter' },
+  { value: 'wordofmouth', label: 'Word of Mouth' },
+  { value: 'youtube', label: 'YouTube' },
+]
+
+const selectStyles = {
+  control: styles => ({
+    ...styles,
+    backgroundColor: 'rgba(236, 236, 236, 0.1)',
+    width: '100%',
+    minHeight: '50px',
+    border: '0',
+    padding: '6px 24px',
+    borderRadius: '4px',
+    marginBottom: '2rem',
+    fontFamily: 'lato',
+    fontWeight: '400',
+    fontSize: '14px',
+    textAlign: 'left',
+  }),
+  placeholder: styles => ({
+    ...styles,
+    fontFamily: 'lato',
+    fontWeight: '400',
+    fontSize: '15px',
+    opacity: '0.3 !important',
+  }),
+  menu: styles => ({
+    ...styles,
+    backgroundColor: 'rgba(32,28,68, 0.8)',
+  }),
+  valueContainer: (styles) => ({
+    ...styles,
+    overflow: 'inherit',
+    padding: '0 0 0 0',
+  }),
+  multiValue: (styles) => ({
+    ...styles,
+    color: '#fff',
+    backgroundColor: 'transparent',
+  }),
+  multiValueLabel: (styles) => ({
+    ...styles,
+    color: '#fff',
+    fontFamily: 'lato',
+    fontWeight: '400',
+    fontSize: '14px',
+    padding: '0 0 0 0',
+  }),
+  multiValueRemove: (styles) => ({
+    ...styles,
+    ':hover': {
+      color:'#aaa',
+      backgroundColor: 'transparent',
+    },
+  }),
+  option: (styles, { isFocused, isSelected }) => {
+    const optionsStyles = {
+      ...styles,
+      ':active': {
+        ...styles,
+        backgroundColor: '#452643',
+      },
+    }
+    if ( isFocused ) {
+      optionsStyles.backgroundColor = '#0a0b24'
+    } else if ( isSelected ) {
+      optionsStyles.color = '#fff'
+      optionsStyles.backgroundColor = 'transparent'
+    }
+    return optionsStyles
+  },
+}
+
+const MultiSelect = (props) => (
+  <Select
+    {...props}
+    isMulti
+    placeholder='What Commit resources have you seen prior to applying?'
+    closeMenuOnSelect={false}
+    options={selectOptions}
+    styles={selectStyles}
```

Comment:
The only prop that should be set at this level is `styles`, everything else depends on the use of the component and might change if forms are added in the future. Might be a good idea to rename the component to simply `Select`.

```js
import ReactSelect from 'react-select'

const Select = (props) => <ReactSelect {...props} styles={selectStyles} />

export default Select
```

This way the `Select` component is just a styled react-select

Reply:
Would this create an issue if we use a styled-component `select`? This multiselect seems quite different from a regular one, but if its the only kind of select we will use then this I can implement this